src/loan.rs

```diff
@@ -445,6 +441,26 @@ impl LoanRequest {
     pub fn collateral_amount(&self) -> Amount {
         self.collateral_amount
     }
+
+    /// Get a reference to the borrower's address.
+    pub fn borrower_address(&self) -> &Address {
+        &self.borrower_address
+    }
+
+    /// Get a reference to the loan borrower's public key.
+    pub fn borrower_pk(&self) -> &PublicKey {
+        &self.borrower_pk
+    }
+
+    /// Get a reference to the fee sats per vbyte.
+    pub fn fee_sats_per_vbyte(&self) -> Amount {
+        self.fee_sats_per_vbyte
+    }
+
+    /// Get a reference to the borrower's collateral inputs.
+    pub fn collateral_inputs(&self) -> &[Input] {
+        self.collateral_inputs.as_slice()
+    }
```

Comment:
If the API using `LoanRequest` is now deprecated, why are are extending the interface of `LoanRequest`? I would - in addition to `interpret`, deprecate the entire `LoanRequest` struct and all functions that deal with it.

Reply:
Yeah, I wasn't sure about this either. I thought this would be a reasonable first step before we get rid of the actor state `struct`s. I should probably already move these to `Borrower0` or get rid of `Borrower0` altogether.

Reply:
> or get rid of Borrower0 altogether.

Deprecate first :)

Reply:
But, don't deprecate until you offer an alternative solution.

Reply:
I've decided against removing any of the actor state `struct`s for now because it is so painful to remodel things when we still have to construct the transactions manually (as opposed to using PSETs).

I will extend `Borrower0` as mentioned above and deprecate `LoanRequest`.

Reply:
Actually annotating `LoanRequest` with `deprecated` creates lots of warnings inside the library because the type still exists. It should be sufficient to mark the APIs that use it as `deprecated`.